app/controllers/thread_controller.php

```diff
@@ -94,4 +98,25 @@ public function write()
         $this->set(get_defined_vars());
         $this->render($page);
     }
+
+    public function delete()
+    {
+        $thread = Thread::get(Param::get('thread_id'));
+        $thread->delete();
+        redirect(url('thread/index'));
+    }
+
+    public function addFollow()
+    {
+        $thread = Thread::get(Param::get('thread_id'));
+        $thread->addFollow();
+        redirect(url('thread/view', array('thread_id' => $_SESSION['thread_id'])));
+    }
+
+    public function removeFollow()
+    {
+        $thread = Thread::get(Param::get('thread_id'));
+        $thread->removeFollow();
+        redirect(url('thread/view', array('thread_id' => $_SESSION['thread_id'])));   
```

Comment:
The same idea in <a href='https://github.com/adrianneperedo/board/pull/2/files#r26107423'>this comment</a> could be applied here po :bow: